database/schema.sql

```diff
@@ -0,0 +1,25 @@
+CREATE DATABASE IF NOT EXISTS fec_reviews;
+
+USE fec_reviews;
+
+CREATE TABLE IF NOT EXISTS reviews (
+  review_id INT unsigned NOT NULL AUTO_INCREMENT,
+  username VARCHAR(150) NOT NULL,
+  fecha DATE NOT NULL,
+  rating INT(5) NOT NULL,
+  title varchar(150) NOT NULL,
+  review_body text NOT NULL,
+  PRIMARY KEY (review_id)
+);
+
+CREATE TABLE IF NOT EXISTS images (
+  image_id INT unsigned NOT NULL AUTO_INCREMENT,
+  image_url VARCHAR(500) NOT NULL,
+  review_id INT unsigned NOT NULL,
+  PRIMARY KEY (image_id),
+  FOREIGN KEY (review_id) REFERENCES reviews(review_id)
+);
+
+/*  Execute this file from the command line by typing:
+ *    mysql -u root < server/schema.sql
+ *  to create the database and the tables.*/
```

Comment:
The comment is a nice touch! Are you planning to proceed without an ORM (like sequelize)? It's optional, but makes certain things a bit simpler to accomplish. Absolutely no problem if you aren't going to use one, just thought I'd bring it up.

Reply:
Haha I mentioned this in a comment to you in one of my code reviews. I'm going to stick with generic MySQL for now and will refactor later if I have time and want to practice it. Thank you.